pokemon-api/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/enums/ChallengeStatus.java

```diff
@@ -0,0 +1,4 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.enums;
+
+public class ChallengeStatus {
+}
```

Comment:
I think we can get away with moving the enums from -persistence to -api instead of reimplementing in both packages.